graph-service/curator.py

```
@@ -0,0 +1,565 @@
+                logger.debug(f"Error checking duplicates: {e}")
+            
+    except Exception as e:
+        logger.error(f"An error occurred during curation run: {e}", exc_info=True)
+    finally:
+        if driver:
+            driver.close()
+
+def main():
+    # Schedule to run every 6 hours
+    schedule.every(6).hours.do(run_curation)
+    logger.info("Curator started – scheduled to run every 6 hours")
```

Ruff (0.14.7)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)